proposals/sar_backscatter.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "sar_backscatter",
+    "summary": "Computes backscatter from georeferenced SAR input",
+    "description": "Computes backscatter from georeferenced SAR input.\n\nNote that backscatter computation may require instrument specific metadata that is tightly coupled to the original SAR products. As a result, this process may only work in combination with loading data from specific collections, not with general data cubes.",
+    "categories": [
+        "cubes",
+        "sar"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "Data cube containing raw backscatter input.",
+            "name": "data",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "description": "The backscatter coefficient to compute, either `sigma0` or `gamma0` (the default).",
+            "name": "backscatter_coefficient",
+            "optional": true,
+            "default": "gamma0",
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "sigma0",
+                    "gamma0"
+                ]
+            }
+        },
+        {
+            "description": "By default, orthorectification is disabled. Set to `true` to enables orthorectification.\n\nThe non-orthorectified products use a simple earth model as provided in the products themselves. This may be sufficient for very flat target areas and is faster to process.",
+            "name": "orthorectify",
+            "optional": true,
+            "default": false,
+            "schema": {
+                "type": "boolean"
+            }
+        },
+        {
+            "description": "The (collection) name of the digital elevation model to use, leave empty to allow the back-end to make a suitable choice. Not all back-ends may offer all digital elevation models. Elevation models are listed as collections, or can be found in back-end specific documentation.",
```

Comment:
Hmm... in openEO I'd expect this to be a load_collection that is then passed to the method. If this really should be just the collection identifier, we should add `"subtype": "collection-id"` (and split the schemas) so that client can help with choosing the id.

Reply:
This option is fairly complicated:
- our sentinelhub backed backscatter offers different DEM's, that we do not have as a collection
- our orfeo based implementation will have access to the DEM's on DIAS, these could also be offered as collection, but orfeo has a DEM loading mechanism of its own

@lforesta  how will it be on EODC?

Reply:
A schema could also be something like:

```json
[
  {
    "type": "string",
    "subtype": "collection-id"
  },
  {
    "type": "string",
    "enum": [
      "dem1",
      "dem2"
    ]
  },
  {
    "type": null
  }
]
```

Then a back-end can choose. What to implement, but that may decrease portability. The best (general-purpose) approach would be "load_collection" (subtype: collection-id).

Reply:
Done.